lib/cc/cli/runner.rb

```diff
@@ -56,6 +59,18 @@ def command
           command_name
         end
       end
+
+      private
+
+      def check_version?
+        first_arg = ARGV.first
+        if %w[--check-version].include? first_arg
```

Comment:
Just curious, is there a reason for the array inclusion instead of `if ARGV.first == "--check-version"`?